### PR TITLE
Add missing delete adstudy helper

### PR DESCRIPTION
**Pull request by michaelwclark**
Untested, but 99% sure this will work. In response to #79.

### CODE REVIEW

File: src/objects/ad-study.js [hunk 1]
```diff
@@ -80,6 +80,12 @@ export default class AdStudy extends AbstractCrudObject {
     );
   }
 
+  deleteAdStudy(params): AbstractObject {
```
Comment (CarlousF) [inline src/objects/ad-study.js]: @michaelwclark Functionality wise this worked for me. Could we rename the function from `deleteAdStudy` to `delete` to be more inline with the deletion functions used for other objects?

Comment (michaelwclark) [reply src/objects/ad-study.js]: @CarlousF this has been pushed

**Comment by jingping2015** [comment]
Close it since we have delete method for `AdStudy`

**Comment by michaelwclark** [comment]
@jingping2015 I'm confused why you closed #79 when this hasn't been merged? Did it get added in in another PR behind the scenes?

**Pull request closed without merging**